### PR TITLE
Jw color field

### PR DESCRIPTION
**Pull request by 0xStarcat**
Deletes `shared/node_modules` directory. (how did that get in??)

- Adds `shared/components/ColorField` component 
- Re-builds `page-component-builder` to use shared component
- Adds new `jw-color-picker` extension which uses the shared component
- Adds some docs

### CODE REVIEW

File: jw-color-picker/src/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
```
Comment (0xStarcat) [inline jw-color-picker/src/index.js:1]: the jw-color-picker index file

File: page-component-builder/src/components/ComponentEditor/index.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import ShortTextField from '../fields/ShortTextField';
 import MultiTextField from '../fields/MultiTextField';
 import LongTextField from '../fields/LongTextField';
 import MarkdownField from '../fields/MarkdownField';
-import ColorField from '../fields/ColorField';
+import ColorField from '@shared/components/ColorField';
```
Comment (0xStarcat) [inline page-component-builder/src/components/ComponentEditor/index.js:19]: switching the imports from the old color field to the shared color field

File: shared/constants/index.js [hunk 3]
```diff
@@ -56,6 +56,166 @@ export const SITE_ROUTING = {
   "Landing Page": "/lp",
 };
 
+
+export const JMS__TLC_COLORS = [
```
Comment (0xStarcat) [inline shared/constants/index.js:60]: new color token constants added. Should be kept in sync with the tokens in the app when possible

**Review by ChrisLTD — DISMISSED** [review]
gtm

**Review by ChrisLTD — APPROVED** [review]
gtm

**Pull request merged**